src/core_modules/capture-core/components/WorkingLists/WorkingListsBase/TemplateMaintenance/SharingDialog.component.js

```diff
@@ -32,7 +37,7 @@ const SharingDialogPlain = ({ onClose, open, templateId, classes }: Props) => {
             open={open}
             id={templateId}
             onRequestClose={handleClose}
-            type={'eventFilter'}
+            type={SHARING_DIALOG_TYPE[storeId] || ''}
```

Comment:
hey @simonadomnisoru  `storeId` is not optional so I think we don't need the fall back in this case?